lib/commands/xctest.js

```diff
@@ -0,0 +1,87 @@
+import B from 'bluebird';
+import { logger } from 'appium-support';
+import log from '../logger';
+import appManagement from './app-management';
+
+let helpers = {}, commands = {}, extensions = {};
+
+const xctestLog = logger.getLogger('XCTest');
+
+function assertIDB (context) {
+  if (!context.opts?.device?.idb) {
+    log.errorAndThrow(`To use XCTest runner, IDB (https://github.com/facebook/idb) must be installed ` +
+      `and sessions must be run with the "launchWithIDB" capability`);
+  }
+}
+
+/**
+ * Run an XCTest
+ *
+ * @param {object} params - The parameters that get passed to IDB to run XC test
```

Comment:
Is `typedef` better for options?
https://github.com/appium/appium-xcuitest-driver/blob/cd9d051652683997413a18b5229446223fa14b7c/lib/commands/biometric.js#L9-L20